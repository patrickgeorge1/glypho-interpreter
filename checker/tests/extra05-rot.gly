+++TyyyM+++)Q1QQw\\\a)))SNNN