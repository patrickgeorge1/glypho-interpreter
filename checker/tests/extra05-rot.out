10000000000000565483421089453189742389541967423619854983447
324656423465647143521534632371342653774239487734
18941654654651034894123198567489743219854987462341456356655406657846479536
